code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -237,14 +237,14 @@ var/const/NO_EMAG_ACT = -50
 	desc = "A silver card which shows honour and dedication."
 	icon_state = MATERIAL_SILVER
 	item_state = "silver_id"
-	job_access_type = /datum/job/hop
```

Comment:
why are these accesses nulled but the useless cards left in?
Can't they be reassigned to new access datums?

Reply:
I'll reassign them and remove some of the no longer used variants.